docs/guide/search/elastic.md

```diff
@@ -0,0 +1,558 @@
+                                        filter: lowercase
+                                        # ...
+      ```
+
+    - `mappings` - Settings under this key define mapping for fields in the index.
+    For more information about mappings, see [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/mapping.html).
+
+        When you create a custom index template, with settings for your own field and document
+        types, make sure that it contains mappings for all searchable fields that are available in eZ Platform.
+        For an example of default configuration with a list of searchable fields, see [Default configuration](https://github.com/ezsystems/ezplatform-elastic-search-engine/blob/v1.0.0/src/bundle/Resources/config/default-config.yaml).
+
+    #### Fine-tune the search results
+
+    Your search results can be adjusted by configuring additional parameters.
+    For a list of available mapping parameters and their usage, see [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/mapping-params.html).
+
+    For example, you can apply a mapping parameter, in this case, a normalizer, to a specific
+    mapping under the `dynamic_templates` key:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    # ...
+                    dynamic_templates:
+                        - ez_string:
+                            match: "*_s"
+                            mapping:
+                            type: keyword
+                            normalizer: lowercase_normalizer
+                        # ...
+    ```
+
+    You can also set a boosting factor for a specific field.
+    Boosting increases the relevance of hits, for example making keywords from the title more relevant than the ones from other places of the document.
+    Set the boosting factor under the `properties` key:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    properties:
+                        content_name_s:
+                            boost: 2.0
+                    # ...
+    ```
+
+    You can even copy contents of existing fields, process them and then paste into another field, which than can be queried:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    properties:
+                        user_first_name_s:
+                            type: keyword
+                            normalizer: lowercase_normalizer
+                            copy_to: custom_field
+                    # ...
+    ```
+
+    ### Bind templates with connections
```

Comment:
IMHO we should link here to ES documentation to explain how it behave in case of multiple templates: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates-v1.html#multiple-templates-v1